natlas-server/app/errors/errors.py

```diff
@@ -1,7 +1,7 @@
 import json
 
 
-class HTTPError:
+class HTTPError(Exception):
```

Comment:
This seems like it would be better called "NatlasServiceError". HTTP is an implementation detail.